.env

```
@@ -5,7 +5,9 @@
 # Your Hypixel secret key, see https://developer.hypixel.net/
 HYPIXEL_SECRET_KEY=
 
-
+# URLs of your 25karma frontend (whitelist)
+# Format: "[url] [url] ..." (space-separated). e.g "https://25karma.xyz http://localhost:3000"
+FRONTEND_URLS=""
```

Can we call this `WHITELISTED_URLS` instead? We can also make it an optional variable - if no URLs are provided then allow all URLs through.
```suggestion
WHITELISTED_URLS=
```

---

Yeah that's better